.github/workflows/release.yml

```diff
@@ -50,11 +50,21 @@ jobs:
           echo 'deb [trusted=yes] https://apt.fury.io/caarlos0/ /' | sudo tee /etc/apt/sources.list.d/caarlos0.list
           sudo apt update
           sudo apt install svu
+      # check if we need to bump tags
+      - name: check for release
+        run: |
+          if [ $(svu next) == $(svu current) ]; then
+            echo "tag match - skip release" &&  echo "skip-release=true" >> $GITHUB_ENV
```

Comment:
I think this is fine, but for ease of reading, can we invert the logic and names and instead do `do-release=false` here and `do-release=true` below on L59?